src/base.jl

```diff
@@ -963,48 +948,44 @@ function get_time_series_values(component::Component, time_series::TimeSeriesDat
     return IS.get_time_series_values(component, time_series)
 end
 
+function get_time_series_keys(component::Component)
+    return IS.get_time_series_keys(component)
+end
+
 """
-Return sorted time series initial times.
+Return the initial times for all forecasts.
 """
-#function get_time_series_initial_times(sys::System)
-#    return IS.get_time_series_initial_times(sys.data)
-#end
+generate_forecast_initial_times(sys::System) = IS.generate_forecast_initial_times(sys.data)
 
 """
-Return an iterable of NamedTuple keys for time series stored for this component.
+Return the total period covered by all forecasts.
 """
-function get_time_series_keys(component::Component)
-    return IS.get_time_series_keys(component)
-end
+get_forecast_total_period(sys::System) = IS.get_forecast_total_period(sys.data)
 
-# TODO 1.0: decide what to do
 """
-Return the horizon for all time series.
+Return the window count for all forecasts.
 """
-function get_time_series_horizon(sys::System)
-    return IS.get_time_series_horizon(sys.data)
-end
+get_forecast_window_count(sys::System) = IS.get_forecast_window_count(sys.data)
 
 """
-Return the earliest initial_time for a time series.
+Return the horizon for all forecasts.
 """
-function get_time_series_initial_time(sys::System)
-    return IS.get_time_series_initial_time(sys.data)
-end
+get_forecast_horizon(sys::System) = IS.get_forecast_horizon(sys.data)
```

Comment:
What's the return in the cases where the is not stored forecasts?

Reply:
0